resources/README_old.md

```diff
@@ -0,0 +1,215 @@
+# 8-ID-I XPCS Bluesky Instrument
```

Comment:
What files are necessary to keep in this folder?  That (and the importance of this folder) could be stated in a `README.md` file.

Reply:
this folder as far as I'm concerned can be deleted. Just kept it around in case you wanted to bring back anything from the old deployment